product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java

```diff
@@ -200,17 +202,24 @@ private void deleteRoles() throws APIManagementException {
     }
 
     @BeforeClass(alwaysRun = true)
-    public void init() throws APIManagerIntegrationTestException, APIManagementException {
+    public void init() throws APIManagerIntegrationTestException, APIManagementException, IOException {
         setupUserData();
         apiStore = new APIStoreRestClient(storeURL);
         apiStore.login(SUPER_USER, SUPER_USER_LOGIN_PW);
         apiPublisher = new APIPublisherRestClient(publisherURL);
         apiPublisher.login(SUPER_USER, SUPER_USER_LOGIN_PW);
         apiPublisherAdmin = new APIPublisherRestClient(publisherURL);
         apiPublisherAdmin.login(ADMIN_LOGIN_USERNAME, ADMIN_LOGIN_PW);
+        // create and publish sample API
+        String swaggerFilePath = System.getProperty("user.dir") + File.separator + "src" + File.separator + "test" +
+                File.separator + "resources" + File.separator + "swaggerFiles" +
+                File.separator + "APIScopeTest1.json";
+
+        File swaggerFile = new File(swaggerFilePath);
+        String swaggerContent = readFromFile(swaggerFile.getAbsolutePath());
```

Comment:
Is this using ScenarioTestUtils.readFromFile ?

Reply:
Yes